playground/backend/internal/code_processing/code_processing.go

```diff
@@ -54,12 +60,14 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 	errorChannel := make(chan error, 1)
 	successChannel := make(chan bool, 1)
 	cancelChannel := make(chan bool, 1)
+	stopReadLogsChannel := make(chan bool, 1)
+	finishReadLogsChannel := make(chan bool, 1)
 
 	go cancelCheck(ctxWithTimeout, pipelineId, cancelChannel, cacheService)
 
 	executorBuilder, err := builder.SetupExecutorBuilder(lc.GetAbsoluteSourceFilePath(), lc.GetAbsoluteBaseFolderPath(), lc.GetAbsoluteExecutableFilePath(), sdkEnv)
 	if err != nil {
-		processSetupError(err, pipelineId, cacheService, ctxWithTimeout)
+		_ = processSetupError(err, pipelineId, cacheService, ctxWithTimeout)
```

Comment:
Wouldn't it be better to process error instead of ignoring it?

Reply:
`processSetupError()` could return an error only if some error happens during write status to cache:
```
func processSetupError(...) error {
        logger.Errorf(...)
	if err = utils.SetToCache(...); err != nil {
		return err
	}
	return nil
}
```
All error processing is happening in the `utils.SetToCache` methods. So we do not need this error and we could use `_` here. 

Moreover on the next line (71) this method `retrun`s.